src/test/pybind/test_rados.py

```diff
@@ -715,6 +715,8 @@ def test_xattrs_op(self):
             self.ioctx.operate_write_op(write_op, 'abc')
 
     def test_locator(self):
+        if os.getenv("CRIMSON_COMPAT") != None:
+            return
```

Comment:
reuse the macro `SKIP_IF_CRIMSON`

Reply:
> reuse the macro `SKIP_IF_CRIMSON`

?I don't understand, `SKIP_IF_CRIMSON` is a c++ macro

Reply:
Ah, not aware that it's python...